### PR TITLE
update to exec contact page

### PR DESCRIPTION
**Pull request by Mole1424**
Updated welfare officer
removed erroneous </a> tag
Exec table now changes to landscape for mobile view

### CODE REVIEW

**Comment by Mole1424** [comment]
* exec table changes to **portrait** for mobile view

File: templates/contact.html [hunk 1]
```diff
@@ -57,6 +57,20 @@
   code.discord {
     cursor: pointer;
   }
+
+  @media screen and (max-width: 487px) {
```
Comment (ericthelemur) [inline templates/contact.html:61]:
```suggestion
  @media screen and (max-width: 487px) {
```
Best to use bootstrap breakpoints for consistency (ensures layouts break at the same point)

https://getbootstrap.com/docs/5.3/layout/breakpoints/#media-queries

**Review by ericthelemur — COMMENTED** [review]
Nearly good, will test a little later this evening

**Comment by Mole1424** [comment]
or could drop publicity a line to make a welfare icon but provisionally removes broken image

**Pull request merged**